app/controllers/storage_locations_controller.rb

```diff
@@ -18,9 +18,16 @@ def index
       @storage_locations = @storage_locations.kept
     end
 
+    inventory_item_names = []
+    InventoryItem.all.each{ |inv_item|
+      cur_item = Item.find(inv_item.item_id)
+      inventory_item_names << cur_item.name if cur_item.active && !inventory_item_names.include?(cur_item.name)
+    }
+    inventory_item_names.sort!{ |a, b| a <=> b }
```

Comment:
This is the default sort, so you don't need to specify any function here. `sort!` will do fine.